test.js

```diff
@@ -130,16 +134,82 @@ test('stripFinalNewline: false', async t => {
 	t.is(stdout, 'foo\n');
 });
 
-test('stripFinalNewline on failure', async t => {
-	const {stderr} = await t.throwsAsync(execa('noop-throw', ['foo'], {stripFinalNewline: true}));
-	t.is(stderr, 'foo');
+test('execa() with .kill after it with SIGKILL should kill cleanly', async t => {
+	const subprocess = execa('node', ['fixtures/no-killable'], {
+		stdio: ['ipc']
+	});
+
+	await waitForProcessLaunch(subprocess);
```

Comment:
`waitForProcessLaunch()` is now simple enough that it could be inlined in the test function, i.e. directly:

```js
await pEvent(subprocess, 'message')
```

Reply:
Done: d793df20c3e1ca071510849b295f16257306ee1e